x/tradeshield/types/events.go

```diff
@@ -7,8 +7,8 @@ import (
 )
 
 const (
-	TypeEvtCloseSpotOrder      = "close_spot_order"
-	TypeEvtClosePerpetualOrder = "close_perpetual_order"
+	TypeEvtCloseSpotOrder       = "close_spot_order"
+	TypeEvtCancelPerpetualOrder = "cancel_perpetual_order"
```

Comment:
@amityadav0 can you clarify why we renaming the close perpetual order to cancel perpetual order?

Reply:
to avoid confusion, so we can close perpetual order which is already open

but this event is used  in cancelling a perpetual open order request. It is not used in closing a perpetual position function

does this make sense ? Can change it something else too ?

Reply:
they are two separate flows:

1) cancel order: to remove an existing perpertual order from the order book (open or close order), the user decides to cancel a pending order basically

2) close order: to execute the closure of an open perpetual position at a given price point or at current market price (limit close or market close)

given this knowledge, can you clarify which one are you trying to implement here?

Reply:
1. cancel order, the above event was use in cancel order as a close order event

Reply:
https://github.com/elys-network/elys/blob/a644435ff2a6155c2b084114c4113bba6e0561a2/x/tradeshield/keeper/msg_server_perpetual_order.go#L126